src/hotspot/share/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -32,19 +32,19 @@
 // declarations and a few frequently used utility functions.
 
 # include <ctype.h>
-# include <string.h>
 # include <stdarg.h>
-# include <stdlib.h>
 # include <stdint.h>
+# include <stdlib.h>
+# include <string.h>
 # include <stddef.h>// for offsetof
 # include <sys/stat.h>
 # include <io.h>    // for stream.cpp
 # include <float.h> // for _isnan
 # include <stdio.h> // for va_list
-# include <time.h>
 # include <fcntl.h>
-# include <limits.h>
 # include <inttypes.h>
+# include <limits.h>
+# include <time.h>
 // Need this on windows to get the math constants (e.g., M_PI).
 #define _USE_MATH_DEFINES
 # include <math.h>
```

Comment:
There's a separate problem, unrelated to include order cleanup, that `_USE_MATH_DEFINES` is being
defined here, as that's too late if `<math.h>` happened to be included already. I'll file a bug to address that.
(Apparently not causing problems currently, but include reorderings could trigger problems.)